stdlib/printexc.ml

```diff
@@ -143,8 +144,8 @@ let format_backtrace_slot pos slot =
       else
         Some (sprintf "%s unknown location" (info false))
   | Known_location l ->
-      Some (sprintf "%s file \"%s\"%s, line %d, characters %d-%d"
-              (info l.is_raise) l.filename
+      Some (sprintf "%s %s in file \"%s\"%s, line %d, characters %d-%d"
```

Comment:
This looks like it will emit a double space if the name is empty.  Same comment for the version in `runtime`.

Reply:
Fixed by ensuring names are never empty.